api/MouseEvent.json

```diff
@@ -1024,26 +1024,36 @@
                 }
               ]
             },
-            "firefox": {
-              "version_added": "30",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "canvas.hitregions.enabled",
-                  "value_to_set": "true"
-                }
-              ]
-            },
-            "firefox_android": {
-              "version_added": "30",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "canvas.hitregions.enabled",
-                  "value_to_set": "true"
-                }
-              ]
-            },
+            "firefox": [
+              {
+                "version_added": "32"
```

Comment:
This is api.MouseEvent.region and appears to be a tricky case. The pref "canvas.hitregions.enabled" actually isn't in [`MouseEvent.webidl`](https://github.com/mozilla/gecko-dev/blob/af6b24b75f82e702476056d6822cb4a2bcf1f985/dom/webidl/MouseEvent.webidl) any longer, but it *is* in [`CanvasRenderingContext2D.webidl`](https://github.com/mozilla/gecko-dev/blob/af6b24b75f82e702476056d6822cb4a2bcf1f985/dom/webidl/CanvasRenderingContext2D.webidl#L333-L338) and seems to be off by default.

I strongly suspect that `region` is always the empty string without the pref enabled, and that we should mark this as `partial_support` or just pretend it's not there and keep it as false.

Can you break out these changes to a separate PR so it doesn't block the rest of this PR?